instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```
@@ -0,0 +1,263 @@
+      ExchangesTotal:
+        metric: exchange
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges, passed or failed, that the selected processor has processed since processor start-up or the last reset operation.
+      FailuresHandled:
+        metric: exchange.failed.handled
+        unit: "{exchanges}"
+        type: counter
+        desc: Indicates the number of exchanges failed and handled by an ExceptionHandler in the context.
+      ExternalRedeliveries:
+        metric: exchange.redelivered.external
+        type: counter
+        unit: "{exchanges}"
+        desc: The total number of all external initiated redeliveries (such as from JMS broker) since processor start-up or the last reset operation.
+      Redeliveries:
+        metric: exchange.redelivered
+        type: counter
+        unit: "{exchanges}"
+        desc: Number of exchanges redelivered (internal only)  since selected processor start-up or the last reset operation.
+      MaxProcessingTime:
+        metric: exchange.processing.max_time
+        unit: ms
+        type: gauge
+        desc: Indicates the longest time, in milliseconds, to process an exchange since processor start-up or the last reset operation.
+      MeanProcessingTime:
+        metric: exchange.processing.mean_time
+        type: gauge
+        unit: ms
+        desc: Indicates the mean processing time, in milliseconds, for all exchanges processed since processor start-up or the last reset operation.
+      MinProcessingTime:
+        metric: exchange.processing.min_time
+        type: gauge
+        unit: ms
+        desc: Indicates the shortest time, in milliseconds, to process an exchange since processor start-up or the last reset operation.
+      LastProcessingTime:
+        metric: exchange.processing.last_time
+        type: gauge
+        unit: ms
+        desc: Indicates the time, in milliseconds, it took the selected processor to process the last exchange.
+      DeltaProcessingTime:
+        metric: exchange.processing.delta_time
+        type: gauge
+        unit: ms
+        desc: Indicates the difference, in milliseconds, of the Processing Time of the last two exchanges transited the selected processor.
+      TotalProcessingTime:
+        metric: exchange.processing.time
+        type: counter
+        unit: ms
+        desc: Indicates the total processing time, in milliseconds, to process all exchanges since start-up or the last reset operation.
+
+
+
+  - bean: org.apache.camel:context=*,type=threadpools,name=*
+    prefix: camel.threadpool.
+    metricAttribute:
+      route: beanattr(RouteId)
+      context: param(context)
+    mapping:
+      ActiveCount:
+        metric: active.count
+        type: updowncounter
+        unit: "{threads}"
+        desc: The approximate number of threads that are actively executing tasks.
+      CompletedTaskCount:
+        metric: completed.task.count
```

(similar to removing `.count` from other counters)
```suggestion
        metric: task.completed
```